packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -20,16 +20,43 @@
   @include min-width('desktop') {
     padding: 28px 48px 48px;
   }
+  .pay-with-heading {
+    color: rgb(115, 115, 115);
+    display: grid;
+    width: 100%;
+    align-items: center;
+    text-align: center;
+    grid-template-columns: minmax(20px, 1fr) auto minmax(20px, 1fr);
+    grid-gap: 20px;
+    &:before,
+    &:after {
+      content: '';
+      border-top: 1px solid #e0e0e6;
+    }
+  }
+
+
+  .subscription-create-pay-with-other .paypal-button {
+    width: 120px;
+    height: 35px;
+    margin: auto; 
+    position: relative;
+      div {
+        // This is very hacky, but without it the button is off-center
+        position: absolute;
+        right: -8px;
```

Comment:
this is gross but something like it is necessary because otherwise the button looks just slightly off center - presumably because it's actually an iframe inside this div.